File: src/nl/rubensten/texifyidea/action/insert/InsertOverlineAction.java
package nl.rubensten.texifyidea.action.insert;

import nl.rubensten.texifyidea.TexifyIcons;
import nl.rubensten.texifyidea.action.InsertEditorAction;

/**
 * @author Ruben Schellekens
 */
public class InsertOverlineAction extends InsertEditorAction {

    public InsertOverlineAction() {
        super("Overline", TexifyIcons.FONT_OVERLINE, "\\overline{", "}");
    }
}

File: src/nl/hannahsten/texifyidea/action/insert/InsertOverlineAction.kt
package nl.hannahsten.texifyidea.action.insert;

import nl.hannahsten.texifyidea.TexifyIcons;
import nl.hannahsten.texifyidea.action.InsertEditorAction;

/**
 * @author Hannah Schellekens
 */
public class InsertOverlineAction extends InsertEditorAction {

    public InsertOverlineAction() {
        super("Overline", TexifyIcons.FONT_OVERLINE, "\\overline{", "}");
    }
}
